### ISSUE TITLE
Support launch type pure JUnit?

### ISSUE THREAD

**Issue by frankbenoit** [ssi-schaefer/lcdsl#51]
We do make a difference between out JUnit plugin tests vs our pure unit tests.
The plugin test take significant more startup time, so we want to run the pure unit test much more often.

Could LCDSL also support this?

**Comment by mduft** [comment]
Yeah, definitely. Plain JUnit support needs some work - it depends all a little on available time (none now unfortunately) and/or contributions which are very welcome.
